CoreFoundation/Locale.subproj/CFCalendar_Internal.h

```diff
@@ -93,10 +93,10 @@ CF_PRIVATE void __CFCalendarZapCal(CFCalendarRef calendar);
 
 CF_PRIVATE CFCalendarRef _CFCalendarCreateCopy(CFAllocatorRef allocator, CFCalendarRef calendar);
 
-CF_PRIVATE Boolean _CFCalendarComposeAbsoluteTimeV(CFCalendarRef calendar, /* out */ CFAbsoluteTime *atp, const char *componentDesc, int32_t *vector, int32_t count);
-CF_PRIVATE Boolean _CFCalendarDecomposeAbsoluteTimeV(CFCalendarRef calendar, CFAbsoluteTime at, const char *componentDesc, int32_t *_Nonnull * _Nonnull vector, int32_t count);
-CF_PRIVATE Boolean _CFCalendarAddComponentsV(CFCalendarRef calendar, /* inout */ CFAbsoluteTime *atp, CFOptionFlags options, const char *componentDesc, int32_t *vector, int32_t count);
-CF_PRIVATE Boolean _CFCalendarGetComponentDifferenceV(CFCalendarRef calendar, CFAbsoluteTime startingAT, CFAbsoluteTime resultAT, CFOptionFlags options, const char *componentDesc, int32_t *_Nonnull * _Nonnull vector, int32_t count);
+CF_EXPORT Boolean _CFCalendarComposeAbsoluteTimeV(CFCalendarRef calendar, /* out */ CFAbsoluteTime *atp, const char *componentDesc, int32_t *vector, int32_t count);
```

Comment:
Why this?

I can’t export these symbols on Darwin. If it is incorrect that thy be private, they should be `CF_CROSS_PLATFORM_EXPORT`.

Reply:
(Is there any way to achieve parity such that s-c-f can use these symbols on Windows as it can on Linux?)

Reply:
These are mismatches between the declarations in the different headers.  Yes, the attempt to get the parity was not well received (the hidden visibility build would do that).  I think that we can do that change separately.

Reply:
I think we may have needed these in the ObjC `NSCalendar.m`, so the cross-platform export would make sense here.

Reply:
Splintered into #1800 with the change to `CF_CROSS_PLATFORM_EXPORT`.